Tests/BaseLib/TestFilePathStringManipulation.cpp

```diff
@@ -15,32 +15,33 @@
 
 #include "BaseLib/FileTools.h"
 
+// TODO: remove as this does not work with Unity builds?
```

Comment:
Can this be updated? Is there any reason why this does not work with Unity builds? Maybe we can also replace the tested functionality with Boost (in another PR)?

Reply:
I removed it completely: you cannot include a cpp in a cpp with Unity builds. File path manipulation methods will be completely removed in subsequent PR as this can be implemented with `std::filesystem`.

Reply:
Nice.